app/src/main/java/com/example/sharewhatyoucanproject/dashboard/DashboardFragment.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.sharewhatyoucanproject.dashboard
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.sharewhatyoucanproject.databinding.FragmentDashboardBinding
+
+class DashboardFragment : Fragment()
+{
+private var _binding: FragmentDashboardBinding? = null
+private val binding get() = _binding!!
+
+override fun onCreateView(
+    inflater: LayoutInflater, container: ViewGroup?,
+    savedInstanceState: Bundle?,
+): View? {
+    _binding = FragmentDashboardBinding.inflate(inflater, container, false)
```

Comment:
binding is inflated this way in fragments so that you can dispose of the binding in `onDestroyView`. Make sure to null this out:

```
override fun onDestroyView() {
  // super call
  _binding = null
}
```

Reply:
done